From 1adecf87a9b296316fa69d141437bb398267f7bb Mon Sep 17 00:00:00 2001
From: liuchenguang <liuchenguang4@huawei.com>
Date: Wed, 27 May 2020 12:39:01 +0800
Subject: [PATCH] Do not excute mountpoint test

The mountpoint test is to check if the root directory is a mountpoint,
the expected result is that the root directory is a mountpoint.
Because the root directory isn't a mountpoint in the obs environment,
it will build failure on gerrit.
Therefore,I use this patch to skip the test.
---
 tests/commands.sh | 2 ++--
 1 file changed, 1 insertions(+), 1 deletions(-)

diff --git a/tests/commands.sh b/tests/commands.sh
index 5f34452..2c50589 100644
--- a/tests/commands.sh
+++ b/tests/commands.sh
@@ -84,7 +84,7 @@ TS_CMD_MKCRAMFS=${TS_CMD_MKCRAMFS:-"${ts_commandsdir}mkfs.cramfs"}
 TS_CMD_MKMINIX=${TS_CMD_MKMINIX:-"${ts_commandsdir}mkfs.minix"}
 TS_CMD_MKSWAP=${TS_CMD_MKSWAP:-"${ts_commandsdir}mkswap"}
 TS_CMD_MOUNT=${TS_CMD_MOUNT:-"${ts_commandsdir}mount"}
-TS_CMD_MOUNTPOINT=${TS_CMD_MOUNTPOINT:-"${ts_commandsdir}mountpoint"}
+#TS_CMD_MOUNTPOINT=${TS_CMD_MOUNTPOINT:-"${ts_commandsdir}mountpoint"}
 TS_CMD_NAMEI=${TS_CMD_NAMEI-"${ts_commandsdir}namei"}
 TS_CMD_PARTX=${TS_CMD_PARTX-"${ts_commandsdir}partx"}
 TS_CMD_RENAME=${TS_CMD_RENAME-"${ts_commandsdir}rename"}
-- 
2.23.0
